src/databricks/labs/ucx/assessment/crawlers.py

```diff
@@ -78,6 +98,32 @@ def _assess_clusters(self, all_clusters):
                 for value in cluster.spark_conf.values():
                     if "dbfs:/mnt" in value or "/dbfs/mnt" in value:
```

Comment:
You're missing a failure check here.

Reply:
Means check for keys in conf

Reply:
Yes, it is checked here -
# Checking if Azure cluster config is present in spark config
                for key, value in cluster.spark_conf.items():
                    for conf in _AZURE_SP_CLUSTER_CONF:
                        if re.search(conf, key):
                            failures.append(f"Uses azure service principal credentials in cluster config.")